mlflow/server/js/src/model-registry/components/ModelView.js

```diff
@@ -111,6 +117,28 @@ export class ModelView extends React.Component {
       });
   };
 
+  // rowSelection object indicates the need for row selection
```

Comment:
It might be cleaner/more in line with the pattern of other components to simply define `onChange` in this component, and have the component that ultimately passes `onChange` to `antd` handle putting it in an antd-friendly object. That is, we'd pass `onChange` down to `ModelVersionTable`, which would then construct a rowSelection object [here](https://github.com/mlflow/mlflow/pull/2537/files#diff-e2845e7c548f5af185e3761f056eec94R90)